basis/http/client/client-tests.factor

```diff
@@ -27,6 +28,7 @@ IN: http.client.tests
 {
     T{ request
         { url T{ url { protocol "https" } { host "www.amazon.com" } { port 443 } { path "/index.html" } } }
+        { proxy-url T{ url } }
```

Comment:
No string reasons...
- to be like the "url" slot (but url is mandatory and this is optional)
- So that f can mean "no proxy" (but the empty URL could mean no proxy)
- so that the behavior "requests use all values from the variables by default, but fill in if it's partial" translatés directly to call derive-url on this slot (but the